stubs/jsonschema/jsonschema/_format.pyi

```diff
@@ -16,28 +20,28 @@ draft7_format_checker: FormatChecker
 draft201909_format_checker: FormatChecker
 draft202012_format_checker: FormatChecker
 
-def is_email(instance) -> bool: ...
-def is_ipv4(instance) -> bool: ...
-def is_ipv6(instance) -> bool: ...
+def is_email(instance: Any) -> bool: ...
```

Comment:
I'll switch this.

Aside/tangent: using `object` in similar functions (e.g. `ensure_is_foo(x: object) -> Foo`) can be less nice than `Any` because of how it responds to type narrowing. After you write enough "assert-is-X" utilities, it becomes a habit to use `Any` in basically any case in which one could use `object`. This is all off-topic, but I wonder if changing the way that `object` narrows would make it more comfortable/normal for typing users like myself to use it. When I first ran into this, I found it _very_ surprising that `isinstance(x, str)` does not narrow `x: object` to `str`.

Reply:
> When I first ran into this, I found it _very_ surprising that `isinstance(x, str)` does not narrow `x: object` to `str`.

That's not true:

https://mypy-play.net/?mypy=latest&python=3.10&gist=5104b5bedec0b6dd05bd8b6cf23fc1ef

Reply:
Oh. Hmm... I know I've run into unexpected issues in which `object` doesn't narrow` but `Any` does. But clearly it wasn't such a simple case. If I see this again, I'll make sure to pay closer attention.

Sorry for the inaccuracy and distraction.